tests/unit_tests/db/test_graphql_plugin_config_repository.py

```
@@ -0,0 +1,42 @@
+@pytest.fixture
+def mock_client() -> Mock:
+    return Mock()
+
+
+@pytest.fixture
+def repository(mock_client: Mock) -> GraphQLPluginConfigRepository:
+    with patch("platzky.db.graphql_plugin_config_repository.Client", return_value=mock_client):
+        repo = GraphQLPluginConfigRepository(
+            "https://test.endpoint", "test_token"
+        )  # NOSONAR - hardcoded token acceptable in tests
+        repo._client  # type: ignore[reportPrivateUsage] # trigger lazy construction now, while Client is patched
```

Ruff (0.16.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)